src/commands/serve.rs

```diff
@@ -76,30 +77,53 @@ pub(crate) fn serve(args: &ArgMatches) {
 }
 
 fn serve_foreground(args: &ArgMatches) {
-    info!("Starting PHP...");
+    let port = find_available_port(parse_default_port(args.value_of("port").unwrap_or(DEFAULT_PORT), DEFAULT_PORT));
+
+    let mut document_root = get_document_root(args.value_of("document-root").unwrap_or("").to_string());
+    if document_root.ends_with('/') { document_root.pop(); }
+    if document_root.ends_with('\\') { document_root.pop(); }
+    document_root.push_str(if cfg!(target_family = "windows") { "\\" } else { "/" });
+    let doc_root_path = PathBuf::from(document_root.as_str());
+    let scripts = vec!["index.php", "app_dev.php", "app.php"];
+    let script_filename = if args.is_present("passthru") {
+        args.value_of("passthru").unwrap_or("index.php").to_string()
+    } else {
+        for script in scripts {
+            let php_entrypoint_path = doc_root_path.join(script);
+            if php_entrypoint_path.is_file() {
+                script.to_string();
```

Comment:
Shouldn't this `;` be removed?

Reply:
Nope, the code don't build without this semi-colon.